webpack/assets/javascripts/foreman_navigation.test.js

```diff
@@ -69,21 +70,13 @@ describe('initNavigation', () => {
          <a href="#" class="dropdown-toggle nav-item-iconic" data-toggle="dropdown" >
            Org1 <span class="caret"></span>
          </a>
-       </li>         
+       </li>
        ${menuWithoutTaxonomies}
      </div>`;
   it('Mark current location/organization as active in vertical menu', () => {
     init();
-    expect(
-      $('.nav-pf-secondary-nav .list-group-item:contains("Loc1")').hasClass(
```

Comment:
Any idea why this gets changed? Afair this is already a pritter output?

Reply:
Yeah, it's changed because the eslint rule for max-len defined [here](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L179) in the airbnb-base config says that strings are ignored. And since there are strings in this line it is ignored. I don't really agree with that setup but that's why we should get pretty integrated into our eslint config.